src/main/kotlin/com/hedvig/accountservice/schedulers/domain/commands/UnableToRequestChargeCommand.kt

```diff
@@ -0,0 +1,11 @@
+package com.hedvig.accountservice.schedulers.domain.commands
+
+import org.axonframework.commandhandling.TargetAggregateIdentifier
+import javax.money.MonetaryAmount
+
+data class UnableToRequestChargeCommand(
+    @TargetAggregateIdentifier
+    val memberId: String,
+    val amount: MonetaryAmount,
+    val scheduledBy: String
+)
```

Comment:
If you want to log it
```suggestion
data class UnableToRequestChargeCommand(
    @TargetAggregateIdentifier
    val memberId: String,
    val amount: MonetaryAmount,
    val scheduledBy: String,
    val exception: Exception
)
```